# HG changeset patch
# User Brian <runningnak3d@gmail.com>
# Date 1278056929 21600
# Branch trunk
# Node ID 1d0142b2583cc1c3c9569c7ad35827f272f603a7
# Parent  456c03cadeb3e5f53de107cdffc8539de82a4371
* Updated DBC file checks for 3.3.5a

diff -r 456c03cadeb3 -r 1d0142b2583c src/server/game/DataStores/DBCStores.cpp
--- a/src/server/game/DataStores/DBCStores.cpp	Thu Jul 01 09:06:56 2010 -0600
+++ b/src/server/game/DataStores/DBCStores.cpp	Fri Jul 02 01:48:49 2010 -0600
@@ -592,13 +592,13 @@
     }
 
     // Check loaded DBC files proper version
-    if (!sAreaStore.LookupEntry(3617)              ||       // last area (areaflag) added in 3.3.3a
-        !sCharTitlesStore.LookupEntry(177)         ||       // last char title added in 3.3.3a
-        !sGemPropertiesStore.LookupEntry(1629)     ||       // last added spell in 3.3.3a
-        !sItemStore.LookupEntry(54860)             ||       // last gem property added in 3.3.3a
-        !sItemExtendedCostStore.LookupEntry(2997)  ||       // last item extended cost added in 3.3.3a
-        !sMapStore.LookupEntry(724)                ||       // last map added in 3.3.3a
-        !sSpellStore.LookupEntry(76567)            )        // last client known item added in 3.3.3a
+    if (!sAreaStore.LookupEntry(3617)              ||       // last area (areaflag) added in 3.3.5a
+        !sCharTitlesStore.LookupEntry(177)         ||       // last char title added in 3.3.5a
+        !sGemPropertiesStore.LookupEntry(1629)     ||       // last added spell in 3.3.5a
+        !sItemStore.LookupEntry(56806)             ||       // last gem property added in 3.3.5a
+        !sItemExtendedCostStore.LookupEntry(2997)  ||       // last item extended cost added in 3.3.5a
+        !sMapStore.LookupEntry(724)                ||       // last map added in 3.3.5a
+        !sSpellStore.LookupEntry(80864)            )        // last client known item added in 3.3.5a
     {
         sLog.outError("\nYou have _outdated_ DBC files. Please extract correct versions from current using client.");
         exit(1);
